Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -146,8 +153,20 @@ class TabTrayViewController: UIViewController,
     }()
 
     private lazy var bottomToolbarItemsForSync: [UIBarButtonItem] = {
-        return [flexibleSpace, syncTabButton]
+        if hasSyncableAccount {
+            return [flexibleSpace, syncTabButton]
+        } else {
+            return []
+        }
     }()
```

Comment:
nit: we could use guard 
```suggestion
        guard hasSyncableAccount else { return [] }

        return [flexibleSpace, syncTabButton]
    }()
```